lib/database_connection.rb

```diff
@@ -0,0 +1,27 @@
+require 'pg'
```

Comment:
This reminds me of the repository pattern.  If you're not already familiar with it, check it out!